cax/tasks/filesystem.py

```diff
@@ -7,11 +7,58 @@
 import os
 import shutil
 import logging
+import subprocess
 
 from cax import config
 from cax.task import Task
 import os
 
+class SetPermission(Task):
+    """Set the correct permissions"""
+    
+    def __init__(self):
+      self.counter = 0
+      self.chmod_file = '/cfs/klemming/projects/xenon/misc/basic_file'
+      self.chmod_foder = '/cfs/klemming/projects/xenon/misc/basic'
+            
+      Task.__init__(self)
+      self.destination_config = config.get_config( config.get_hostname() )
+
+    def go(self):
+      """Run the standard procedure to set ownership and permissons for files/folders"""
+      self.ChangePermisson()
+      self.SetCounter()
+      self.ResetCounter(60) #Reset counter after 60 cycles: Permissions and ownership is changed every hour
+
+    def ChangePermisson(self):
+      set_rec = "-R"
+      
+      if self.counter == 0 and config.get_hostname() == "tegner-login-1":
```

Comment:
I'd remove counter.  Also, just do if hostname is not tegner, return.  It's clearer.